libcxx/docs/index.rst

```diff
@@ -65,7 +65,17 @@ Getting Started with libc++
 
 Current Status
 ==============
+libc++ has become the default C++ Standard Library implementation for many major platforms, including Apple's macOS, iOS,
+watchOS, and tvOS, as well as Google's Android operating system. As a result, libc++ has an estimated user base of
+over 1 billion daily active users, making it one of the most widely-deployed C++ Standard Libraries in the world.
```

Comment:
Let's mention FreeBSD too, it has a lower user base but still would be nice to mention them.

Reply:
Done.